prefect_databricks/flows.py

```diff
@@ -430,14 +438,21 @@ async def jobs_runs_wait_for_completion(
         def jobs_runs_wait_for_completion_flow():
             databricks_credentials = DatabricksCredentials.load("BLOCK_NAME")
             return jobs_runs_wait_for_completion(
-                multi_task_jobs_run_id=45429,
+                multi_task_jobs_runs_id=45429,
                 databricks_credentials=databricks_credentials,
                 run_name="my_run_name",
                 max_wait_seconds=1800,  # 30 minutes
                 poll_frequency_seconds=120,  # 2 minutes
             )
         ```
     """
+    warnings.warn(
+        "The flow `jobs_runs_submit_and_wait_for_completion` will be moved to"
+        " prefect_databricks.jobs.flows in the next release, but can be imported with"
+        " `from prefect_databricks.jobs import"
+        " jobs_runs_submit_and_wait_for_completion`",
+        DeprecationWarning,
+    )
```

Comment:
I think this is a little more explicit.
```suggestion
        "jobs_runs_wait_for_completion has been moved to the "
        "prefect_databricks.jobs module. jobs_runs_wait_for_completion "
        "is available from prefect_databricks.flows for backwards compatibility, but will "
        "be removed in a subsequent release. Please update your code to import ",
        "jobs_runs_wait_for_completion from prefect_databricks.jobs."
```